src/Elcodi/Component/Geo/View/Address.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Geo\View;
+
+use DateTime;
+
+use Elcodi\Component\Geo\Entity\Interfaces\AddressInterface;
+use Elcodi\Component\Geo\Services\Interfaces\LocationProviderInterface;
+use Elcodi\Component\Geo\ValueObject\LocationData;
+use Elcodi\Component\Geo\View\Interfaces\AddressInterface as AddressViewInterface;
+
+/**
+ * Class Address
+ */
+class Address implements AddressViewInterface
+{
+    /**
+     * @var array
+     *
+     * The hierarchies already retrieved.
+     */
+    private static $hierarchies = [];
+
+    /**
+     * @var LocationProviderInterface
+     *
+     * A location provider interface.
+     */
+    private $locationProvider;
+
+    /**
+     * @var AddressInterface
+     *
+     * An address.
+     */
+    private $address;
+
+    /**
+     * Builds a new view.
+     *
+     * @param AddressInterface          $address          The address to
+     *                                                    convert to a view.
+     * @param LocationProviderInterface $locationProvider A location provider
+     */
+    public function __construct(
+        AddressInterface $address,
+        LocationProviderInterface $locationProvider
+    ) {
+        $this->address = $address;
+        $this->locationProvider = $locationProvider;
+    }
+
+    /**
+     * Get id
+     *
+     * @return string Id
+     */
+    public function getId()
+    {
+        return $this->address->getId();
+    }
+
+    /**
+     * Get Address
+     *
+     * @return mixed Address
+     */
+    public function getAddress()
+    {
+        return $this->address->getAddress();
+    }
+
+    /**
+     * Get AddressMore
+     *
+     * @return mixed AddressMore
+     */
+    public function getAddressMore()
+    {
+        return $this->address->getAddressMore();
+    }
+
+    /**
+     * Get Comments
+     *
+     * @return string Comments
+     */
+    public function getComments()
+    {
+        return $this->address->getComments();
+    }
+
+    /**
+     * Get Mobile
+     *
+     * @return mixed Mobile
+     */
+    public function getMobile()
+    {
+        return $this->address->getMobile();
+    }
+
+    /**
+     * Get Name
+     *
+     * @return mixed Name
+     */
+    public function getName()
+    {
+        return $this->address->getName();
```

Comment:
Sorry, All these methods are copy pasted from the Address entity, I'll change it here.